doc/ledger3.texi

```diff
@@ -4635,13 +4635,16 @@ Ledger will include @samp{; transid: 767718} in the first transaction
 from the file above.
 
 @findex --invert
+@findex --auto-match
 @findex --account @var{STR}
 @findex --rich-data
 
-The @command{convert} command accepts three options. They are
-@option{--invert} which inverts the amount field, @option{--account
-@var{STR}} which you can use to specify the account to balance against,
-and @option{--rich-data} which stores additional metadata as tags.
+The @command{convert} command accepts four options. They are
+@option{--invert} which inverts the amount field, @option{--auto-match}
+which automatically match account from Ledger journal for every CSV
+line, @option{--account @var{STR}} which you can use to specify the
+account to balance against, and @option{--rich-data} which stores
+additional metadata as tags.
```

Comment:
I propose "which stores additional tag/value pairs." Because here tags are associated with a value, I believe this was the meaning of the person who wrote "which stores additional metadata as tags." Because here we are not speaking of tag without data (=metadata), but a tag with metadata (=data). Well, the wording is dangerous I admit, that is why I propose tag/value pair.